build/templates/backend/scripts/base_archive.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+{{define "main" -}}
+
+set -e
+
+GETH_BIN={{.Env.BackendInstallPath}}/{{.Coin.Alias}}/geth
+DATA_DIR={{.Env.BackendDataPath}}/{{.Coin.Alias}}/backend
+
+CHAINDATA_DIR=$DATA_DIR/geth/chaindata
+SNAPSHOT=https://mainnet-archive-snapshots.base.org/$(curl https://mainnet-archive-snapshots.base.org/latest)
```

Comment:
Unfortunately they stopped providing archive snapshots for Base for geth. We would have to switch to reth. But for our use case, it would be enough to download non archive snapshot and run it in archive mode. Geth in that case starts to store the archive data for the new blocks.

Reply:
d6a94e766bd886c35baca0abbdaa1b7c74469f56